teams/index.html

```diff
@@ -169,23 +169,6 @@ <h4 class="text-center">Roger Leigh</h4>
                     </div>
                 </div>
                 <div class="column">
-                    <a class="team-member" data-open="modal9">
-                        <img class="thumbnail" alt="photo of Simone Leo" src="{{ site.baseurl }}/img/team/simone_leo.jpg">
-                        <h5>Simone Leo</h5>
-                        <p class="job-title">Software Developer</p>
-                        <span class="bio-link tiny button hollow">Read Bio</span>
-                    </a>                                                            
-                    <div class="reveal" id="modal9" data-reveal>
-                      <h4 class="text-center">Simone Leo</h4>
-                      <p class="job-title">Software Developer</p>
-                      <hr>
-                      <p>Simone Leo joined the Dundee team in 2015 after being a satellite developer for OME based at CRS4 in Italy. He received an M.Sc. degree in Electronic Engineering from the University of Cagliari, Italy, in 2002 and a PhD in Biomedical Engineering from the same university in 2015. Before joining CRS4 in 2006, he worked in the private sector as a system administrator. His research interests are mainly focused on distributed computing models and applications, with an emphasis on their applicability to computational biology.</p>
-                      <button class="close-button" data-close aria-label="Close reveal" type="button">
-                        <span aria-hidden="true">&times;</span>
-                      </button>
-                    </div>
-                </div>
-                <div class="column">
                     <a class="team-member" data-open="modal10">
```

Comment:
It's okay that these "modal" `data-open` attribute values are no longer contiguously numbered?

Reply:
AFAIK it's fine as long as the ID matches between the team member section and the reveal section. Is that correct @lunson ?

Reply:
That makes sense, I can test on that basis.